src/actions.h

```diff
@@ -118,13 +118,13 @@ class Actions final : public BaseEvents
 		ReturnValue canUse(const Player* player, const Position& pos, const Item* item);
 		ReturnValue canUseFar(const Creature* creature, const Position& toPos, bool checkLineOfSight, bool checkFloor);
 
-		bool registerLuaEvent(Event* event);
+		bool registerLuaEvent(Action* event);
+		void clear(bool fromLua) final;
```

Comment:
You mixed `final` and `override` everywhere. I believe you should use both.

Reply:
to be fair they where like that to begin with, I just moved them from `private:` to `public:` but yea I can change that

Reply:
Oh sure, do it while working around here :smile: